src/index.html

```diff
@@ -17,6 +17,12 @@
     />
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <div class="stopwatch stopwatch--speed-up">
+      <div class="stopwatch__container">
+        <div class="stopwatch__minute"></div>
+        <div class="stopwatch__seconds"></div>
```

Comment:
Class names should follow the BEM methodology as per the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules). The class `stopwatch__seconds` should be `stopwatch__second` to maintain consistency and correct semantics, as each class represents a single second.